Modules/Core/FiniteDifference/include/itkFiniteDifferenceSparseImageFilter.h

```diff
@@ -197,7 +197,9 @@ class ITK_TEMPLATE_EXPORT FiniteDifferenceSparseImageFilter
     FiniteDifferenceSparseImageFilter * Filter;
     TimeStepType                        TimeStep;
     std::vector<TimeStepType>           TimeStepList;
-    std::vector<bool>                   ValidTimeStepList;
+
+  // NB: although semantically boolean, vector<bool> is not thread safe due to the possibility of multiple bits being packed together in the same memory location.
```

Comment:
The line will prevent format check. This one will work:
```cpp
// NB: although semantically boolean, vector<bool> is not thread safe due to the possibility of multiple bits being
// packed together in the same memory location.
```

Reply:
I suspect that Sean has disabled `clang-format` on his computer, possibly accidentally.

Reply:
Xcode annoyingly doesn't provide clang-format.  I should figure out how to install it myself...

Reply:
With VTK one just writes "Do: reformat" in gitlab and an OCD bot makes these kinds of changes. There's nothing similar here?

Reply:
ITK build downloads a version of `clang-format` executable, and then uses it in a pre-commit hook. At least usually. Something is out of order on your computer.

Reply:
Take a look at https://github.com/InsightSoftwareConsortium/ITK/blob/master/CMake/ITKModuleClangFormat.cmake

Reply:
Oh this rings a bell now, I do recall somehow disabling that as it was super annoying just trying to to do local commits of WIP work.

Enforcing white space OCD seems better suited to pre-merge than pre-commit, IMHO.